ops/charm.py

```diff
@@ -1678,7 +1679,7 @@ class RelationMeta:
     ``metadata.yaml`` and used by the charm code if appropriate.
     """
 
-    VALID_SCOPES = ['global', 'container']
+    VALID_SCOPES: ClassVar[List[str]] = ['global', 'container']
```

Comment:
Arguably, it's a constant and could be a tuple or a frozenset instead of a list.
In that case, ruff won't bother us about ClassVar.

My reading is that ruff only insists that **mutable** class-level fields should be ClassVars.

It appears that no charm uses this thing (semantically a constant, but technically a public field on a public class).

Which means that we could change it's type.

cc @tonyandrewmeyer and @benhoyt

Reply:
On the other hand, https://peps.python.org/pep-0526/ states that the intention of ClassVar is to prevent the field being accidentally overridden in an instance, regardless of mutability.